indigo/bulk_creator.py

```diff
@@ -405,7 +411,7 @@ def link_repeal(self, work, info):
         if not repealing_work:
             return self.create_task(work, info, task_type='link-repeal')
 
-        repeal_date = repealing_work.commencement_date
+        repeal_date = repealing_work.main_commencement_date()
```

Comment:
I'm beginning to wonder if we should simply make `commencement_date` an alias for `main_commencement_date()`?